WorkflowUI/Sources/Hosting/WorkflowHostingController.swift

```diff
@@ -18,46 +18,57 @@
 
 import ReactiveSwift
 import UIKit
+@_spi(ViewEnvironmentWiring) import ViewEnvironmentUI
 import Workflow
 
 /// Drives view controllers from a root Workflow.
 public final class WorkflowHostingController<ScreenType, Output>: UIViewController where ScreenType: Screen {
+    public typealias CustomizeEnvironment = (inout ViewEnvironment) -> Void
```

Comment:
should this alias be nested under the generic parent type? the fully-specified spelling will be parametrized by the screen/output, which might be unwieldy. then again, maybe nobody will need to reference it in that manner?

Reply:
Yeah, I'd be surprised if we ever seen folks referencing the type directly in the context of `WorkflowHostingController` usage.

I can move it out if we'd prefer though! There are a few opportunities in Market to re-use this type if it was at the top level. Let me know if you'd prefer we do move it out, and if so, what to name it/where to put it!

Reply:
your call to relocate/rename if you think it makes sense – just a thought!